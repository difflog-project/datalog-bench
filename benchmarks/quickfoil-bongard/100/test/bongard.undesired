1253687
6258160
705277
8774316
3845478
6749196
7827979
5467686
364359
703105
