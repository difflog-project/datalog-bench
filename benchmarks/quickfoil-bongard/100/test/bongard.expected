6803059
1522714
2328356
3067415
5099604
8692901
3935502
389096
8463396
4146961
